vcl_templates/www.vcl.erb

```diff
@@ -256,10 +256,6 @@ sub vcl_recv {
 
 #FASTLY recv
 
-  if (req.request != "HEAD" && req.request != "GET" && req.request != "FASTLYPURGE") {
-    return(pass);
-  }
```

Comment:
This will mean we now lookup every request in the cache, rather than just GET/HEAD.

Reply:
Looks like you've also included purge here that you'd want to keep.

Reply:
I think `req.resuest != "FASTLYPURGE"` here means that `return(pass);` would not be called for purge requests, so removing the whole block won't have an affect. I added in FASTLYPURGE to the check in vcl_fetch, though.

Reply:
Ah sorry my bad